integration-testing/pom.xml

```diff
@@ -75,5 +101,35 @@
             <artifactId>pg-embedded</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mssqlserver</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>no-docker</id>
```

Comment:
At some point, would it make sense to move the `no-docker` concept to be part of each test case (maybe as an assumption, or a tag, or custom extension) rather than having to maintain lists of test cases in pom?

Reply:
no-docker solely exists for you. :-)

Reply:
I still feel it's important to be able to build Jdbi without external daemons.
Docker is even worse, because testcontainers uses ryuk which doesn't work out of the box rootless, so it in fact requires a root daemon to build Jdbi then. (without hacking it to work with podman)